docs/upgrade.html

```diff
@@ -21,6 +21,10 @@
 0.10.0.0 has <a href="#upgrade_10_breaking">potential breaking changes</a> (please review before upgrading) and
 there may be a <a href="#upgrade_10_performance_impact">performance impact during the upgrade</a>. Because new protocols
 are introduced, it is important to upgrade your Kafka clusters before upgrading your clients.
+<p/>
+<b>Notes to client with version 0.9.0.0: </b>Due to a bug introduced in 0.9.0.0,
+clients with ZooKeeper dependency may not be able to work with 0.10.0.0 broker. Therefore, 0.9.0.0 clients should
```

Comment:
@ijuma @allenxwang This is a big deal right? It breaks Kafka's backwards compatibility guarantee. It should probably be moved from the performance impact section to the breaking section. Is there anything else we can do to fix this? Are we sure it does not break 0.8 clients? 

cc @gwenshap @junrao